integration_tests/src/main/python/spark_session.py

```diff
@@ -127,8 +127,8 @@ def with_spark_session(func, conf={}):
     reset_spark_session_conf()
     _add_job_description(conf)
     # Only set the ansi conf if not set by the test explicitly by setting the value in the dict
-    if "spark.sql.ansi.enabled" not in conf and get_per_test_ansi_mode() is not None:
-        conf["spark.sql.ansi.enabled"] = get_per_test_ansi_mode()
+    if "spark.sql.ansi.enabled" not in conf and is_per_test_ansi_mode_disabled() is not None:
+        conf["spark.sql.ansi.enabled"] = is_per_test_ansi_mode_disabled()
```

Comment:
Sorry, sir, but this doesn't look right to me.

If `is_per_test_ansi_mode_disabled` is `true` then `spark.sql.ansi.enabled` needs to be made `false`.

Reply:
Here is my suggestion: 
1. Rename `is_per_test_ansi_mode_disabled` to `is_per_test_ansi_mode_enabled`.
2. Rename the underlying variable from `per_test_ansi_mode` to `per_test_ansi_mode_enabled`. These are both stated in the positive.
3. Negate when assigning to `per_test_ansi_mode_enabled`, at conftest.py:222.